source/includes/_processing.md

```diff
@@ -0,0 +1,314 @@
+# Processing
+
+The OpenDataSoft platform allows you to apply one or more processors to a dataset. These processors are units of data transformation and other processing. Each processor represents a configurable operation that will be applied to all rows of a dataset. Examples of what you can do with a processor include replacing text with a regex, geocoding an address into geographical coordinates, creating a new column that contains the result of a substraction between two existing columns and much more.
```

Comment:
Maybe add a link to the platform documentation for the details of all processors?

Reply:
:+1: